server/pg/sql_analyzer_velox.cpp

```diff
@@ -1312,10 +1316,16 @@ void SqlAnalyzer::ProcessInsertStmt(State& state, const InsertStmt& stmt) {
 
   const auto& table = basics::downCast<catalog::Table>(logical_object);
   const auto& table_type = *table.RowType();
+  size_t table_columns = table_type.size();
+  if (!table.PKColumns().empty() &&
+      table.PKColumns()[0] == catalog::Column::kGeneratedPKId) {
+    table_columns--;
```

Comment:
Is it still possible? I guess only on connector level generated pk would be added in this case.

Reply:
fixed